app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -63,6 +73,9 @@ object Project extends GeoJsonSupport {
         description,
         visibility,
         tileVisibility,
+        isAOIProject,
+        aoiCadence,
+        now,
```

Comment:
`now` might not be the best value for this. The first AOI check wouldn't start until `aoiCadence` milliseconds had passed.